Scripts/common.ps1

```diff
@@ -28,9 +28,17 @@ function Write-Log
   Write-Host "[$timestamp] $message"
 }
 
-function Execute-Command ($exe, $arguments)
+function Execute-Command ($exe, $arguments, $mask)
 {
-  Write-Log "Executing command '$exe $($arguments -join ' ')'"
+  $maskedArgs = $($arguments -join ' ')
+  if ($mask -ne $null) {
```

Comment:
PSScriptAnalyzer would whinge here about putting null on the right when it's better on the left. But as with other comments, it works for now, and we're not currently running PSScriptAnalyzer on it.